lib/fluent/log.rb

```diff
@@ -371,6 +387,13 @@ def configure(conf)
           @log = PluginLogger.new($log)
         end
         @log.level = @log_level
+        @log.optional_header = "[#{self.class.name}#{plugin_id_configured? ? "(" + @id + ")" : ""}] "
+        @log.optional_attrs = {
```

Comment:
Is it needed to set `plugin_type` and `plugin_id` into `optional_attrs`?
This is ignored in actual logging by `unless` condition in L287. What's other purpose of these?